src/Sylius/Bundle/ThemeBundle/composer.json

```diff
@@ -36,14 +36,16 @@
         "phpunit/phpunit": "^4.1",
         "symfony/twig-bundle": "^2.7.7"
     },
-    "minimum-stability": "dev",
-    "prefer-stable": true,
```

Comment:
We do not have ``prefer-stable`` in other bundles and Symfony doesn't use it - https://github.com/symfony/http-foundation/blob/master/composer.json. If we decide it is good idea to add it then we should do it for all bundles and components.